packages/api/src/external/fhir-converter/converter.ts

```diff
@@ -27,7 +27,7 @@ export enum FHIRConverterCDATemplate {
 
 export type ContentMimeType = Pick<Document["content"], "mimeType">;
 
-export function isConvertible(mimeType?: string | undefined): boolean {
+export function isConvertible(mimeType?: string | undefined | null): boolean {
```

Comment:
Adding null here because the schema for docref from CQ is nullish

Reply:
We can keep it as `string | undefined` and fallback to `undefined` if `nullish` on the caller. If we're not careful with these things, soon we'll be writing `undefined | null` on every place that's optional.